packages/shared/src/db/queries.ts

```diff
@@ -1867,6 +1874,32 @@ export const setLeftGroups = createWriteQuery(
   ['groups', 'channels']
 );
 
+// Includes latest post as well as unconfirmed posts
+export const getUnconfirmedPosts = createReadQuery(
```

Comment:
why did I include latest post here?

- When doing `useChannelPosts`, we union these posts alongside the main query so that we always have unconfirmed posts, regardless of post window
- So it looks like `main query` + `unconfirmeds`
- naively, `unconfirmeds` does not include the latest post – this is because we fetch latest post for each channel on init, which means it is no longer unconfirmed!
- to get the expected behavior, just throw the latest post in as well